pysd/py_backend/vensim/vensim2py.py

```diff
@@ -144,12 +144,14 @@ def get_model_elements(model_str):
 
     model_structure_grammar = _include_common_grammar(r"""
     model = (entry / section)+ sketch?
-    entry = element "~" element "~" element ("~" element)? "|"
+    entry = element "~" element "~" doc ("~" element)? "|"
     section = element "~" element "|"
     sketch = ~r".*"  #anything
 
     # Either an escape group, or a character that is not tilde or pipe
     element = ( escape_group / ~r"[^~|]")*
+    # Anything other that is not a tilde or pipe
```

Comment:
@marrobl Are tilde or pipe allowed in Vensim docstrings? Particularly in double-quoted escape group?

Reply:
I have tried to change a comment of teacup model in Vensim. 
I added “example~example” in one comment. Then I saved it and what Vensim did was omit the ~. Also, when you write ~ in a comment, Vensim underlines it in yellow. The same for |.
So, tilde or pipe are not allowed in Vensim docstrings.

Reply:
Perfect. Thanks!